src/lib/infrastructure/controller.py

```diff
@@ -804,3 +806,20 @@ def delete_images(
         )
         use_case.execute()
         return self.response
+
+    def assign_images(self, project: str, image_names: list, user: list):
```

Comment:
delete to # noqa: E203 def assign_images(self, project_name: str, folder_name: strimage_names: list, user: list):